core/client/fs/src/main/java/alluxio/client/file/cache/DefaultMetaStore.java

```diff
@@ -13,29 +13,39 @@
 
 import alluxio.exception.PageNotFoundException;
 
-import java.util.HashSet;
-import java.util.Set;
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * The default implementation of a metadata store for pages stored in cache.
  */
 public class DefaultMetaStore implements MetaStore {
-  private final Set<PageId> mPageMap = new HashSet<>();
+  /** A map from PageId to page info. */
+  private final Map<PageId, PageInfo> mPageMap = new HashMap<>();
 
   @Override
   public boolean hasPage(PageId pageId) {
-    return mPageMap.contains(pageId);
+    return mPageMap.containsKey(pageId);
   }
 
   @Override
-  public boolean addPage(PageId pageId) {
-    return mPageMap.add(pageId);
+  public void addPage(PageId pageId, PageInfo pageInfo) {
```

Comment:
addPage requires PageInfo now.

Reply:
Do we still need to pass in separate page id now that PageInfo has the id?